### PR TITLE
Forgot to update web-fragment

### PR DESCRIPTION
**Pull request by KyleAure**
Forgot to create a 6.1 version of web-fragment in #44

### CODE REVIEW

File: xml/src/web-fragment_6_1.xsds [hunk 1]
```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2009, 2024 Oracle and/or its affiliates and others.
```
Comment (Emily-Jiang) [inline xml/src/web-fragment_6_1.xsds:4]: Is this a correct copyright?

Comment (KyleAure) [reply xml/src/web-fragment_6_1.xsds:4]: I just duplicated the 6_0 version and updated the imports/versions... In that case I would think the copyright history would follow.

Comment (Emily-Jiang) [reply xml/src/web-fragment_6_1.xsds:4]: ah, okay. Thanks for explaining this!

**Comment by KyleAure** [comment]
@starksm64 - review for review.

**Comment by KyleAure** [comment]
Closing duplicate of #48

**Pull request closed without merging**